tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -442,14 +435,30 @@ TRTEngineOp::EngineCtxPair& TRTEngineOp::GetEngine(
   // TODO(tmorris): using first input to get batch size - is this reliable?
   const int batch_size = input_shapes[0].dim_size(0);
 
+  // Get engine cache
+  auto trt_rm = TRTResourceManager::instance();
```

Comment:
Please don't use the resource manager singleton, use `context->resource_manager()` instead.